media-plugins/vdr-xineliboutput/metadata.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<maintainer type="person">
+		<email>hd_brummy@gentoo.org</email>
+		<name>Joerg Bornkessel</name>
```

Comment:
@hd-brummy Are you fine with adding yourself as primary maint?

Reply:
@ConiKost I just copied maintainer order from media-video/vdr where metadata.xml has been written by @hd-brummy  itselt.  So I assume he's fine with that here, too.

Reply:
This make no sense to add me, as the only member in the Gentoo VDR Project its me. All bugs around  to vdr@g.o reach me direct.
A had some request in the past, why iam and vdr@g.o is listed in some metadata.xml.  ;)

Reply:
Thanks. So please drop hd_brummy and leave only the project :-)